### ISSUE TITLE
CommonJS bug: "window.setTimeout is not a function"

### ISSUE THREAD

**Issue by dhritzkiv** [typekit/webfontloader#246]
Using browserify, and the latest version of webfontloader (v1.5.19), I'm having issues. My code:

````javascript
var WebFont = require("webfontloader");

WebFont.load({
	typekit: {
		id: 'xxxxx',
	},
	timeout: 3000,
	active: function() {
		console.log("loaded");
	}
});
````

I get this error when calling `WebFont.load`:

````
Uncaught TypeError: window.setTimeout is not a function
````

Am I doing something incorrectly?

[Here is a requirebin page](http://requirebin.com/?gist=d5a64b7d0c7e727d6506) with the code. (Check the console)

**Comment by bramstein** [comment]
This is fixed in v1.5.21, thanks for reporting it.

**Comment by dhritzkiv** [comment]
Great. Thanks!
